src/traits.rs

```diff
@@ -1454,20 +1447,20 @@ impl NomRange<usize> for Range<usize> {
   }
 
   fn is_inverted(&self) -> bool {
-    !(self.start < self.end)
+    self.start >= self.end
   }
 
   fn saturating_iter(&self) -> Self::Saturating {
     if self.end == 0 {
-      1..0
+      Range::default()
```

Comment:
I'm not sure this will have te exact same behaviour

Reply:
Yes. They're both empty ranges.

[Default range vs. 1..0 range playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=0bb21608cbcfe6ac7eb66359c5714e49)